m4_sessions/1803-inning/group_7.md

```diff
@@ -1,13 +1,48 @@
 Group 7
 
-Group Members:
+Group Members: 
+Tyler Madsen, Lauren Billington, Gabe Afflitto, Max Stackhouse
 
-Name of Session:
+Name of Session: 
+Finding Meaningful Common Ground
 
 Session Description (Why should students come to this session):
+Strategies to finding connections with new people to build closer relationships with others.
```

Comment:
What benefit can this bring to students?  Is it good for networking?  Is it good for building your professional network?  Is it good for life/perspective?